backup/manager_test.go

```diff
@@ -63,13 +65,21 @@ func (m *MockTester) DownloadBackupFiles(nodeID, backupID string) ([]string, err
 	return m.DownloadBackupFilesImpl(nodeID, backupID)
 }
 
+func (m *MockTester) SetTor(torConfig *tor.TorConfig) {
+	m.torConfig = torConfig
+}
+
+func (m *MockTester) TestAuth() {
+	return
```

Comment:
I guess one could add a test but it's unclear how useful it would be. That is, the WebDav provider requires us to test authentication but other providers don't.